src/docs/background/payments-overview/index.md

```diff
@@ -0,0 +1,397 @@
+provider, and the specific types of verification information depends on
+the provider's country. You can check the verification requirements for
+your most likely marketplace provider demographics in
+[Stripe's own documentation](https://stripe.com/docs/connect/required-verification-information).
+
+#### 2. Customer checkout
+
+When the customer initiates a transaction in the Flex default
+transaction processes, Flex creates a
+[PaymentIntent](/background/payment-intents/) for the total price of the
+transaction. Once the PaymentIntent is confirmed, Stripe preauthorizes
+the sum from the customer's payment method. In other words, the sum does
+not leave the customer's account, but it is still reserved and not
+available to be used by the customer.
+
+The preauthorization is valid for 7 days, after which the
+preauthorization is automatically released by Stripe, and the funds are
+again available to the customer. In the FTW templates, creating and
+confirming the PaymentIntent are triggered at the same customer action
+for the default process.
+
+**Related Stripe actions:**
+
+- [:action/stripe-create-payment-intent](/references/transaction-process-actions/#actionstripe-create-payment-intent)
+- [:action/stripe-confirm-payment-intent](/references/transaction-process-actions/#actionstripe-confirm-payment-intent)
+
+#### 3. Provider acceptance
+
+When a customer has requested to book a listing, a provider has 6 days
+to accept the booking until it expires automatically. This timeline
+ensures that the Stripe preauthorization does not expire before the
+provider has the opportunity to accept or reject the booking. Once the
+provider accepts the booking, the PaymentIntent is captured and the
+transaction sum is transferred from the customer's account to the
+platform account.
+
+In the
+[FTW Product (Sneakertime)](https://github.com/sharetribe/ftw-product)
+template, the default process combines customer checkout and provider
+acceptance to all be triggered at the same customer action.
+
+**Related Stripe actions:**
+
+- [:action/stripe-capture-payment-intent](/references/transaction-process-actions/#actionstripe-capture-payment-intent)
+
+#### 4. Customer refund
+
+If the customer requests a refund for one reason or another, the
+operator can refund the PaymentIntent. The Flex integration with Stripe
+only supports full refunds. The default integration takes into account
+whether or not the PaymentIntent has already been captured from the
+customer's account.
+
+**Related Stripe actions:**
+
+- [:action/stripe-refund-payment](/references/transaction-process-actions/#actionstripe-refund-payment)
+
+#### 5. Provider payout
+
+Once the booking has completed successfully, the provider's payout is
+paid to the bank account that is linked to their Custom Connect account.
+Depending on how the
+[transaction's line items](/background/pricing/#line-items) have been
+defined, the platform can take a commission of the price from either the
```

Comment:
Commissions could also be mentioned in step 3. as during the payment intent capture, the commissions are paid to the platform account from the Connect account. The way that it's implemented is that commissions are paid using a Stripe concept called _application fee_. When Flex invokes Stripe to capture a payment intent, the application fee is passed in the request as a param and therefore Stripe knows to transfer the amount to the Connect account and from that deduct the application fee and move that to the platform account.

Reply:
Mention added!